website/alternative_views.py

```diff
@@ -314,3 +321,102 @@ def get_context_data(self, **kwargs):
             .order_by("-total_score")[:10],
         )
         return context
+
+
+class CreateHunt2(TemplateView):
+    model = Hunt
+    fields = ["url", "logo", "domain", "plan", "prize", "txn_id"]
+    template_name = "hunt2.html"
+
+    @method_decorator(login_required)
+    def get(self, request, *args, **kwargs):
+        
+        return render(request, self.template_name)
+        # except Exception as e:
```

Comment:
if this code is no longer used, you can delete it

Reply:
This code is for creating bughunts hunts new page which i will work on after this pr